src/hotspot/share/opto/runtime.cpp

```diff
@@ -192,6 +192,72 @@ bool OptoRuntime::generate(ciEnv* env) {
+const TypeFunc* OptoRuntime::_fast_arraycopy_Type                 = nullptr;
+const TypeFunc* OptoRuntime::_checkcast_arraycopy_Type            = nullptr;
+const TypeFunc* OptoRuntime::_generic_arraycopy_Type              = nullptr;
+const TypeFunc* OptoRuntime::_slow_arraycopy_Type                 = nullptr;
+const TypeFunc* OptoRuntime::_unsafe_setmemory_Type               = nullptr;
+const TypeFunc* OptoRuntime::_array_fill_Type                     = nullptr;
+const TypeFunc* OptoRuntime::_array_sort_Type                     = nullptr;
+const TypeFunc* OptoRuntime::_array_partition_Type                = nullptr;
+const TypeFunc* OptoRuntime::_aescrypt_block_Type                 = nullptr;
+const TypeFunc* OptoRuntime::_cipherBlockChaining_aescrypt_Type   = nullptr;
+const TypeFunc* OptoRuntime::_electronicCodeBook_aescrypt_Type    = nullptr;
+const TypeFunc* OptoRuntime::_counterMode_aescrypt_Type           = nullptr;
+const TypeFunc* OptoRuntime::_galoisCounterMode_aescrypt_Type     = nullptr;
+const TypeFunc* OptoRuntime::_digestBase_implCompress_with_sha3_Type      = nullptr;
+const TypeFunc* OptoRuntime::_digestBase_implCompress_without_sha3_Type   = nullptr;
+const TypeFunc* OptoRuntime::_digestBase_implCompressMB_with_sha3_Type    = nullptr;
+const TypeFunc* OptoRuntime::_digestBase_implCompressMB_without_sha3_Type = nullptr;
+const TypeFunc* OptoRuntime::_multiplyToLen_Type                  = nullptr;
+const TypeFunc* OptoRuntime::_montgomeryMultiply_Type             = nullptr;
+const TypeFunc* OptoRuntime::_montgomerySquare_Type               = nullptr;
+const TypeFunc* OptoRuntime::_squareToLen_Type                    = nullptr;
+const TypeFunc* OptoRuntime::_mulAdd_Type                         = nullptr;
+const TypeFunc* OptoRuntime::_bigIntegerShift_Type                = nullptr;
+const TypeFunc* OptoRuntime::_vectorizedMismatch_Type             = nullptr;
+const TypeFunc* OptoRuntime::_ghash_processBlocks_Type            = nullptr;
+const TypeFunc* OptoRuntime::_chacha20Block_Type                  = nullptr;
+const TypeFunc* OptoRuntime::_base64_encodeBlock_Type             = nullptr;
+const TypeFunc* OptoRuntime::_base64_decodeBlock_Type             = nullptr;
+const TypeFunc* OptoRuntime::_string_IndexOf_Type                 = nullptr;
+const TypeFunc* OptoRuntime::_poly1305_processBlocks_Type         = nullptr;
+const TypeFunc* OptoRuntime::_intpoly_montgomeryMult_P256_Type    = nullptr;
+const TypeFunc* OptoRuntime::_intpoly_assign_Type                 = nullptr;
+const TypeFunc* OptoRuntime::_updateBytesCRC32_Type               = nullptr;
+const TypeFunc* OptoRuntime::_updateBytesCRC32C_Type              = nullptr;
+const TypeFunc* OptoRuntime::_updateBytesAdler32_Type             = nullptr;
+const TypeFunc* OptoRuntime::_osr_end_Type                        = nullptr;
+const TypeFunc* OptoRuntime::_register_finalizer_Type             = nullptr;
+JFR_ONLY(
```

Comment:
For a multi-line case, `#ifdef INCLUDE_JFR` would look better here.

Reply:
updated.